index-about.html

```diff
@@ -0,0 +1,145 @@
+  <link rel="stylesheet" href="./style.css">
+  <title>Alex's Concert Page</title>
+</head>
+<body id="home">
+  <header class="header">
+    <nav class="nav1">
+      <ul>
+          <li class="microo">
+            <a rel="noopener" target="_blank" href="https://web.facebook.com/microverseinc/?_rdc=1&_rdr"><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li class="microo">
+            <a rel="noopener" target="_blank" href="https://twitter.com/microverseinc"><i class="fab fa-twitter"></i></a>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+      </ul>
+  </nav>
+    <nav class="head-all">
+      <i class="fas fa-bars menu"></i>
+      <ul>
+        <li class="logo-img">
+          <a href="./index.html">
+            <img src="./images/logo-music.png" alt="">
+          </a>
+        </li>
+        <li>
+          <a href="./index-about.html"><h2>About</h2></a>
+        </li>
+        <li>
+          <a href="#program"><h2>Program</h2></a>
+        </li>
+        <li>
+          <a href="#"><h2>Guests</h2></a>
+        </li>
+        <li>
+          <a href="#partners"><h2>Partners</h2></a>
+        </li>
+        <li class="campagn">
+          <a href="#"><h2>MC Campaign</h2></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="menu-sec">
+      <nav>
+        <a href="./index.html"><h2 class="links">Home</h2></a>
+        <a href="./index-about.html"><h2 class="links">About</h2></a>
+        <a href="#program"><h2 class="links">Program</h2></a>
+        <a href="#guests"><h2 class="links">Guests</h2></a>
+        <a href="#partners"><h2 class="links">Partners</h2></a>
+      </nav>
+      <div class="cancel links">
+        &times;
+      </div>
+    </section>
+    <article id="about" class="about">
+      <section class="about-sec1">
+        <p class="para1">
+          "Come All Come One"
+        </p>
+        <h1 class="main-title">
+          CULTURAL MUSIC AFRICA CONCERT 2021
+        </h1>
+        <p class="para2">
+          Music, as a cultural right, may aid in the promotion and protection of other human rights. It can help in the healing process, dismantling walls and boundaries, reconciliation, and education. Around the world, music is being used as a vehicle for social change and bringing communities together.
+        </p>
+        <p class="para2">
+          Please contact us per Email for any further questions about MC Africa Concert 2021!
+          <a href="#">
+            <b>
+              mcconcert2021@mckenya.org <span>&GT;</span>
+            </b>
+          </a>
+        </p>
+        <h2 class="about-title">
+          Cultural Music Africa Concert 2021 Logo
+        </h2>
+        <p class="para2">
+          The logo of MC Music Culture 2021 was decided through the logo competition from 9, April to 10 May.
+        </p>
```

Comment:
- [ ] Please add the background image as seen in the original design.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/192045243-a61356c7-2587-404e-88d4-1ac013142e65.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/192045327-4d84ad48-eaf5-4e5d-8ef4-250cfd4a838c.png)

- [ ] Add all the necessary background colours and designs as that of the original design for this block of the about page.